packages/flutter/test/rendering/editable_test.dart

```diff
@@ -406,9 +406,9 @@ void main() {
     expect(
       editable,
       paints
+        ..paragraph()
```

Comment:
You should add at least one new test for this if you can, too.  Maybe can you draw some text with a backgroundColor, then select it, then verify that the selection color isn't covered?